facebookads/api.py

```diff
@@ -65,6 +69,10 @@ def __init__(self, body=None, http_status=None, headers=None, call=None):
         self._headers = headers or {}
         self._call = call
 
+        # ensure all headers keys are upper-case
+        _headers = dict((key.upper(), value) for (key, value) in headers.items())
+        log.info("Rate limit header X-FB-ADS-INSIGHTS-THROTTLE: %s", _headers.get("X-FB-ADS-INSIGHTS-THROTTLE"))
```

Comment:
Since the headers are exposed via `def headers` - can you do this logging within your application code? I wish Facebook SDK provided more tools to work with throttling / usage limitations as well, but current solution seems to be very use case specific.

I'm also not sure if this code would work? `X-FB-ADS-INSIGHTS-THROTTLE` is only relevant to Insights api, while `FacebookResponse` is used for other APIs like Marketing API. I don't think they return this header, so that `.get` would potentially fail?